ntoskrnl/mm/balance.c

```diff
@@ -15,8 +15,6 @@
 #include <debug.h>
 
 #include "ARM3/miarm.h"
```

Comment:
Restoring original newlines
```suggestion
#include "ARM3/miarm.h"


```